<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewTilesetDialog</class>
 <widget class="QDialog" name="NewTilesetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>292</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>818</width>
    <height>292</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>818</width>
    <height>292</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="nameEdit">
               <property name="minimumSize">
                <size>
                 <width>416</width>
                 <height>27</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>416</width>
                 <height>27</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Image File</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="fileEdit">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>386</width>
                 <height>27</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>386</width>
                 <height>27</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="buttonBrowse">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Tile Size    </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="tilesizeEdit">
             <property name="minimumSize">
              <size>
               <width>86</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>86</width>
               <height>27</height>
              </size>
             </property>
             <property name="minimum">
              <number>16</number>
             </property>
             <property name="maximum">
              <number>64</number>
             </property>
             <property name="singleStep">
              <number>16</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGraphicsView" name="tileSetView">
         <property name="minimumSize">
          <size>
           <width>280</width>
           <height>232</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>280</width>
           <height>232</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QPushButton" name="buttonCancel">
         <property name="minimumSize">
          <size>
           <width>196</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonOK">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
